client/src/featureform/sqlite_metadata.py

```diff
@@ -175,9 +175,22 @@ def createTables(self):
         self.__conn.execute('''CREATE TABLE IF NOT EXISTS models(
           name        text PRIMARY KEY NOT NULL,
           type        text,
-          description text,
           status      text);''')
 
+        # models training sets table
+        self.__conn.execute('''CREATE TABLE IF NOT EXISTS model_training_sets(
+          model_name           text NOT NULL,
+          training_set_name    text NOT NULL,
+          training_set_variant text NOT NULL,
+          UNIQUE(model_name, training_set_name, training_set_variant));''')
```

Comment:
Do we have a test for if we register a model to the same feature twice?

Reply:
Yes, we do. In `client/tests/test_serving_model.py`, I've included this test for both training sets and features:
* `test_registering_same_model_twice_while_serving_training_set`
* `test_registering_same_model_twice_while_serving_features`

I haven't, however, included a test for registering the same model to two different features/training sets. I assumed this test would cover that behavior as well, but if you feel otherwise, I can add tests for that case, too.